synapse/lib/types.py

```diff
@@ -821,6 +821,7 @@ def __init__(self, load=True):
 
         self.addTypeCast('country:2:cc', self._castCountry2CC)
         self.addTypeCast('make:guid', self._castMakeGuid)
+        self.addTypeCast('json:dumps', self._castJsonDumps)
```

Comment:
dig this idea.  since "dumps" is a programmery thing, we should make it something like make:json :)

Reply:
done